### ISSUE TITLE
13.1 shader export seems fully correct for the first time ever, however "in Blender" display is ridiculous looking even in comparison to 13.0

### ISSUE THREAD

**Issue by Akira13641** [BadDogSkyrim/PyNifly#184]
Here's a few comparisons of a shell-textured Mihail Cyrodilic wolf:

13.0 (had still-wrong exports of shaders to NIF):
![Nifly130](https://github.com/BadDogSkyrim/PyNifly/assets/11223712/9b6f2ef7-9740-4c64-8ddd-402789a2da5a)

13.1 (totally perfect exports of shaders to NIF, as far as I can tell):
![Nifly131](https://github.com/BadDogSkyrim/PyNifly/assets/11223712/303c119c-e0b5-4c00-9c65-8fa1de96c646)

NifSkope (the closest **by far**  to what the mesh actually looks like in-game in ideal rendering circumstances):
![NifSkope](https://github.com/BadDogSkyrim/PyNifly/assets/11223712/4b419149-2d6c-4473-b3c6-6df7e92a005f)

Nothing in Skyrim ever looks even remotely like the weird melty multi-color thing that 13.1 is doing, basically. Getting closer to NifSkope rendering would be nice, but even just reverting to 13.0 rendering (without losing the correct exports) would be a reasonabe solution, I think.

**Comment by BadDogSkyrim** [comment]
Yeah, I think that was a mistake there. I've been fooling with the port to Blender 4.0, which has me reworking the entire shader setup. You can certainly break the Metallic input and it will all work and that should improve the look some right away.

I've also discovered that I haven't been turning on subsurface at all and that makes a bunch of difference with human heads.

If you want to shoot me your nif I'll see what I can see. It _ought_ to be possible to do anything the game does with blender's shader, but this is not my forte.

**Comment by BadDogSkyrim** [comment]
Don't bother with the nif, I loaded up the vanilla werewolf and OMG. I think I'll have to look at the "Skyrim Shader Type" and use different values depending on that.

**Comment by jjtnb123** [comment]
hi bro，can you tell me how to use this plugin.I've never succeeded

**Comment by jjtnb123** [comment]
when i import hkx ，It always goes wrong
